public/app/plugins/datasource/cloudwatch/components/LogsQueryEditor.tsx

```diff
@@ -12,15 +12,15 @@ import { CloudWatchLanguageProvider } from '../language_provider';
 import CloudWatchLink from './CloudWatchLink';
 import { css } from 'emotion';
 
-type Props = QueryEditorProps<CloudWatchDatasource, CloudWatchQuery>;
+type Props = QueryEditorProps<CloudWatchDatasource, CloudWatchQuery> & { allowCustomValue?: boolean };
```

Comment:
Nitpick: we probably do not need this as a prop if we never set it to false.

Reply:
Should probably make it more explicit but it's false by omission in Explore, since template variables aren't supported there